resources/views/components/input/file.blade.php

```diff
@@ -0,0 +1,57 @@
+@props([
+    'name',
+    'label',
+    'value' => null,
+    'error',
+    'informationLabel' => 'Information',
```

Comment:
```suggestion
    'informationLabel' => null,
```

not a very informative default label, i would just make it optional